src/Agent.Listener/Configuration/LinuxServiceControlManager.cs

```diff
@@ -100,37 +100,31 @@ public override void StartService(string serviceName)
                 ChangeOwnershipToLoginUser(filesToChange);
                 ReloadSystemd();
                 ExecuteSystemdCommand("start " + serviceName);
+                _term.WriteLine(StringUtil.Loc("ServiceStartedSuccessfully", serviceName));
             }
             catch (Exception)
             {
-                _term.WriteError(StringUtil.Loc("LinuxServiceStartFailed"));
+                _term.WriteError(StringUtil.Loc("CanNotStartService"));
                 throw;
             }
         }
 
-        protected virtual string GetUnitFile(string serviceName)
+        public override void StopService(string serviceName)
         {
-            return Path.Combine(SystemdPathPrefix, serviceName);
-        }
-
-        protected virtual bool CheckIfSystemdExists()
-        {
-            Trace.Info(nameof(CheckIfSystemdExists));
+            Trace.Info(nameof(StopService));
```

Comment:
Default log level is info. Configuration should log most things (everything?) as info, but once the agent starts listening we don't necessarily want to log everything as info. Otherwise we end up with really noisy logs. Outside of configuration, the plan is to at least log high level things to info and optionally be able to flip on higher verbosity if we really need noisy logs to investigate a specific issue.